src/entity.js

```diff
@@ -788,7 +788,7 @@ Entity.removeTable = function() {
  * Create an entity on azure table with property and mapping.
  * Returns a promise for an instance of `this` (typically an Entity subclass)
  */
-Entity.create = function(properties, overwriteIfExists) {
+Entity.create = async function(properties, overwriteIfExists) {
```

Comment:
`Entity.create` doesn't seem to use the `await` keyword. Is this needed?

Reply:
It is -- I explained in the commit comment.  An async function will *always* return a Promise, even if some synchronous function it calls throws an error.  Without this, `create` sometimes throws an error and sometimes returns a rejected Promise.

Reply:
Got it. Makes sense. Just wanted to double check :)